tests/stdlib/tsugar.nim

```diff
@@ -1,4 +1,72 @@
-import sugar
+discard """
+  output: '''
+x + y = 30
+'''
+"""
+
+import std/sugar
+
+# `=>`
+
+block:
```

Comment:
group lexically, eg:
```nim
block: # `=>`
  block:
    ...
  block:
     ...
block: # `->`
  ...
```
makes it easier to follow and search for tests for a given proc

Reply:
Do you mean for all procs? E.g. for `collect`, there are quite a lot of tests.

Reply:
at least the ones you're adding/editing would be nice